docs/contributing.rst

```diff
@@ -30,8 +30,7 @@ Team Calls
 If you have issues or pull requests to discuss, or are interested in hearing what
 the team and contributors are working on, you can join our public team calls:
 
-- Mondays at 3pm CET/CEST.
-- Wednesdays at 2pm CET/CEST.
+- Mondays and Wednesdays at 3pm CET/CEST.
 
 Both calls take place on `Jitsi <https://meet.ethereum.org/solidity>`_.
```

Comment:
I think the link should be updated too, otherwise, people will join an empty room.